ProjFS.Mac/PrjFSLib/PrjFSUser.cpp

```diff
@@ -70,7 +70,7 @@ io_connect_t PrjFSService_ConnectToDriver(enum PrjFSServiceUserClientType client
         return IO_OBJECT_NULL;
     }
     
-    if (!PrjFSService_ValidateVersion(prjfsService))
+    if (performVersionValidation && !PrjFSService_ValidateVersion(prjfsService))
```

Comment:
@pmj thinking about this more it looks like the "ConnectToDriver" function is also used when we start the virtualization instance.  Will we fail to mount if there is a version mismatch?

Reply:
Yes, if there's a version mismatch, 

    s_kernelServiceConnection = PrjFSService_ConnectToDriver(UserClientType_Provider);

will fail, as `performVersionValidation` defaults to `true` in this change. So no change of behaviour there.

Reply:
Cool, I am still happy to take this change.  It's minimal and the right place to fail would be on setting the virtualization up.